R/model.R

```diff
@@ -1493,9 +1493,10 @@ artnum_mf <- function(calendar_quarter, art_number, naomi_mf) {
     }
 
     if(!is.null(art_number)) {
-      ## No number on ART data and no year specified
+      ## Number on ART data available but no year
       raw_input  <- aggregate_art(art_number, naomi_mf$areas) %>%
         dplyr::mutate(
+          art_current = art_current_adjusted,
           source = "programme",
```

Comment:
@jeffeaton this is where reported ART is being replaced with adjusted ART for model fitting.